Jaewook/add hoth xi winner

dcho-jaewook opened the pull request:
Winner.jsx
- added an empty array for the HOTH-XI winning projects (line 7)
- modified const winners to include HOTH-XI (line 488)

Gallery.jsx
- Included menu item for HOTH XI (line 27)

jamesmwu commented:
In the future, please include the issue which you are fixing in the description of your PR.

Fixes #437

jamesmwu reviewed (changes requested):
1. Please remove the commit related to the Apply button, since that is for a different PR.
2. You should resolve your conflicts with main. Do this by checking out the main branch, pulling, then switching back to this branch, and running `git merge main`. Resolve any merge conflicts as necessary.
3. When someone opens the Gallery page, the first thing they should see is the HOTH XI screen. Currently, it is still showing the HOTH X screen when initially navigated to.
4. You must add images for the winners. Reach out to @melissaqshi for information about where to find and access images from HOTH X.

jamesmwu commented:
If you must prioritize, please focus on your Apply button PR first before looking into this one :)

Thanks Jaewook!

jamesmwu reviewed (changes requested):
Discussed in person regarding changes, but please make sure the Gallery properly displays all the winners and their projects. Additionally, please ensure that all previous changes requested are addressed so we can avoid repeating the same feedback. I believe there are also conflicts with the main branch at the moment.

Reach out to @hannahkendall04 if you encounter any issues with the photo carousel failing to display!

dcho-jaewook commented:
**Gallery.jsx**
- Fixed `year` from X to XI
- Added XI for the menu item

**gallery-winners/hoth-XI & gallery-carousel/hothXI**
- Added images for the winner projects
- Added images for the HOTH XI gallery

**Winners.jsx**
- Added new array `hothXI` to store information about HOTH XI winner projects

**PhotoCarousel.jsx**
- Imported HOTH XI images
- Created carousel array for the case where `year` === 'XI'

jamesmwu commented on src/components/Gallery/PhotoCarousel.jsx:
```diff
@@ -33,6 +33,16 @@ import hothX_image3 from '../../images/gallery-carousel/hoth-X/image3.jpg';
 import hothX_image4 from '../../images/gallery-carousel/hoth-X/image4.jpg';
 import hothX_image5 from '../../images/gallery-carousel/hoth-X/image5.jpg';
 
+import hothXI_image1 from '../../images/gallery-carousel/hoth-XI/image1.jpg';
+import hothXI_image2 from '../../images/gallery-carousel/hoth-XI/image2.jpg';
+import hothXI_image3 from '../../images/gallery-carousel/hoth-XI/image3.jpg';
+import hothXI_image4 from '../../images/gallery-carousel/hoth-XI/image4.jpg';
+import hothXI_image5 from '../../images/gallery-carousel/hoth-XI/image5.jpg';
+import hothXI_image6 from '../../images/gallery-carousel/hoth-XI/image6.jpg';
+import hothXI_image7 from '../../images/gallery-carousel/hoth-XI/image7.jpg';
+import hothXI_image8 from '../../images/gallery-carousel/hoth-XI/image8.jpg';
+import hothXI_image9 from '../../images/gallery-carousel/hoth-XI/image9.jpg';
+
```
In the photo carousel here, I think we only need to display the pictures of the HOTH XI winners-- so it's probably OK to remove the ones that feature just the officers or anything that isn't the winners holding their certificates.

jamesmwu reviewed (changes requested):
After the requested changes are addressed, I think this LGTM!

Seems like the devpost links don't work at the moment but I don't think that's your fault, I think devpost is down right now? At least I can't connect to it even when I search devpost.com. Thanks Jaewook!

jamesmwu reviewed (approved):
Awesome, LGTM! Thanks Jaewook!

jamesmwu commented:
@hannahkendall04 Please review when you have a moment as well :)

hannahkendall04 reviewed (approved):
LGTM

jamesmwu commented:
> LGTM

Just for future reference, if you're the last reviewer you can feel free to just squash and merge into main! I've taken care of it for this PR 👍

hannahkendall04 commented:
> > LGTM
> 
> Just for future reference, if you're the last reviewer you can feel free to just squash and merge into main! I've taken care of it for this PR 👍

Ohh good to know thank you!! Will def do that going forward!
